static/json/languages/ar_AR.json

```diff
@@ -522,8 +522,8 @@
         "import_wallet": "إضافة, استعادة سرية, عبارة البذور, المفتاح الخاص",
         "watch_wallet": "إضافة",
         "add_hardware_wallet": "ledger, trezor, الاتصال",
-        "hide_balances": "تبديل",
-        "hide_small_balances": "تبديل",
+        "hide_balances": "التبديل، اخفاء، اظهار، الأرصدة، الكشف",
```

Comment:
Noticed an issue with commas on some of these will will cause a parsing issue; adding a rule to Crowdin to use proper `,` for all of these strings